### DISCUSSION TITLE
How about actual professional achievements?

### DISCUSSION THREAD

**Discussion by boxcee** [community/community#18650] [Other Feature Feedback, Questions, & Ideas]
I don't have a strong opinion on the new achievements. However, I could see them showing my CKA certificate for example. Or my GCA cert. Any chance that would be possible?

**Comment by 5HT2** [comment]
No, this doesn't seem like it will be added.

If you'd _like_, you can add a README (a markdown document) to the top of your profile page, like so: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes
